### DISCUSSION TITLE
what happened to the past branches?

### DISCUSSION THREAD

**Discussion by fdefilippo** [minio/minio#14139] [Q&A] [answered]
hi all, yesterday all the branches of the old releases were removed from git, why?

best,

fd.

**Comment by harshavardhana** [comment, marked as answer]
Those branches are meant for our customers. Those branches have been moved over to relevant downstream customer repositories.

You shouldn't be relying on them @fdefilippo

**Comment by fdefilippo** [comment]
So we can't access anymore?

**Reply by harshavardhana** [reply]
> So we can't access anymore?

You can access relevant releases here https://github.com/minio/minio/releases
